.ci/test-r-package.sh

```diff
@@ -19,7 +19,7 @@ if [[ $R_BUILD_TYPE != "cran" ]]; then
 fi
 
 # Get details needed for installing R components
-R_MAJOR_VERSION=( ${R_VERSION//./ } )
+R_MAJOR_VERSION="${R_VERSION%.*.*}"
```

Comment:
```suggestion
R_MAJOR_VERSION="${R_VERSION%.*}"
```

I forgot that for these CI jobs, we just use `{major}.{minor}` for this `R_VERSION` variable.

https://github.com/microsoft/LightGBM/blob/41ba9e8f00c89d72e5cb71c964722ce1ed4d8445/.github/workflows/r_package.yml#L108

https://github.com/microsoft/LightGBM/blob/41ba9e8f00c89d72e5cb71c964722ce1ed4d8445/.github/workflows/r_package.yml#L209